annotation/app/models.py

```diff
@@ -97,6 +102,13 @@ class AnnotatedDoc(Base):
     tasks = relationship("ManualAnnotationTask", back_populates="docs")
 
 
+def default_tree(column_name: str) -> Callable:
+    def default_function(context) -> str:
+        return Ltree(f"{context.current_parameters.get(column_name)}")
```

Comment:
If we get None, should it be "None"(str)?

Reply:
default tree depends on default id which can not be not. But yes, this is an implicit behaviour